torchvision/transforms/transforms.py

```diff
@@ -739,7 +739,7 @@ class RandomResizedCrop(torch.nn.Module):
             int instead of sequence like (h, w), a square output size ``(size, size)`` is
             made. If provided a tuple or list of length 1, it will be interpreted as (size[0], size[0]).
         scale (tuple of float): range of size of the origin size cropped
```

Comment:
Could you please also rewrod the docstring for scale.

Reply:
done!